TradeItIosTicketSDK2/TradeItFxTradingTicketViewController.swift

```diff
@@ -66,6 +66,46 @@ class TradeItFxTradingTicketViewController: TradeItViewController, UITableViewDa
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+
+        let isChosenAccountEnabled = self.order.linkedBrokerAccount?.isEnabled ?? false
+        if (!isChosenAccountEnabled) {
+            self.showAccountSelection()
+            return
+        }
+
+        // Check if selected account supports trading FX
+        TradeItSDK.linkedBrokerManager.getAvailableBrokers(
```

Comment:
Is it possible to use the orderCapabilities on the linkedBrokerAccount ?
```swift
guard let  _ = self.order.linkedBrokerAccount?.orderCapabilities(forInstrument: TradeItTradeInstrumentType.fx)` else {
//alert
}
```

Reply:
I don't want to switch to that yet because it's not yet supported by the Linked Broker sync functionality.

Reply:
Also I just looked at the server and we haven't even implemented this for equities!

Reply:
It should be implemented in the account response for both. 
The second endpoint `getFxOrderCapabilities` is when it depends on the ticker. Indeed there is no implementation for equities for that case.

Reply:
For equity, we should be able to do the check here: https://github.com/tradingticket/TradeItIosTicketSDK2/blob/ac2f996e98e2f0bec69614cfc9575fc494dc4838/TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift#L313

As we authenticateIfNeeded when we load the ticket, we should get the order capabilities, even if it is not in the cache ?